webapp/src/ts/services/contact-save.service.ts

```diff
@@ -55,6 +56,10 @@ export class ContactSaveService {
     }
 
     const doc = this.prepare(submitted.doc);
+    if (typeof original === 'undefined' && doc.should_create_user === 'true') {
```

Comment:
Not a big deal, but from the perspective of naming consistency, would it make sense to change `should_create_user` to `create_user_for_contact`?